eo-runtime/pom.xml

```diff
@@ -80,7 +80,7 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>@{argLine} -Duser.language=ru -Duser.region=RU</argLine>
+          <argLine>@{argLine} -Duser.language=ru -Duser.region=RU -Ddataized.log.until=5</argLine>
```

Comment:
@mximp what is this for? typo?

Reply:
@yegor256 Ah. That is part of dataization logging functionality which I opted not to include. 
Thanks for spotting.